frontend/next.config.mjs

```diff
@@ -15,4 +16,49 @@ const nextConfig = {
   },
 };
 
+if (!process.env.BABEL_ENV || process.env.BABEL_ENV !== "coverage") {
+  // it seems that sentry and babel are not compatible but fortunately we need
```

Comment:
I'd be very very surprised if they weren't.
There's even this: https://www.npmjs.com/package/@sentry/babel-plugin-component-annotate